### DISCUSSION TITLE
请问下各位大佬 formily 在开发环境如何实现代码热重载

### DISCUSSION THREAD

**Discussion by Sun79** [alibaba/formily#4135] [Q&A 2.x]
当前在 Vue 中使用时，修改相同 name 的 Field 上的 props 时，视图并不会更新，必须手动刷新页面才能生效，是否有好的方法在开发环境直接热重载页面。

**Comment by coolbob1998** [comment]
Formily 状态基本都是全局的，开发环境热更新不太好搞
